ibazel/command/default_command.go

```diff
@@ -43,21 +45,22 @@ func DefaultCommand(startupArgs []string, bazelArgs []string, target string, arg
 }
 
 func (c *defaultCommand) Terminate() {
-	if c.pg != nil && !subprocessRunning(c.pg.RootProcess()) {
+	if !c.IsSubprocessRunning() {
+		c.pg = nil
 		return
 	}
-
-	// Kill it with fire by sending SIGKILL to the process PID which should
-	// propagate down to any subprocesses in the PGID (Process Group ID). To
-	// send to the PGID, send the signal to the negative of the process PID.
-	// Normally I would do this by calling c.cmd.Process.Signal, but that
-	// only goes to the PID not the PGID.
-	c.pg.Kill()
-	c.pg.Wait()
-	c.pg.Close()
+	c.termSync.Do(func() {
+		terminate(c.pg)
+	})
 	c.pg = nil
 }
 
+func (c *defaultCommand) SendKillSignal() {
```

Comment:
Nit: `s/SendKillSignal/Kill`. This will read as `command.Kill()`which I believe is more descriptive because in windows it is not sending a kill signal.